skyreader/plot/plot.py

```diff
@@ -617,67 +903,122 @@ def bounding_box(ra, dec, theta, phi):
-                    [1., 2.1459/1.177], ['m', 'm'], ['-', '--']):
-                spline_contour = self.circular_contour(extra_ra_rad, extra_dec_rad,
-                    extra_radius_rad*cont_scale, healpy.get_nside(equatorial_map))
+            healpy.projscatter(
+                np.degrees(extra_lon),
+                np.degrees(extra_lat),
+                lonlat=True,
+                c='m',
+                marker='x',
+                s=20,
+                label=r'Reported online (50%, 90%)'
+            )
+            for cont_scale, cont_col, cont_sty in zip(
+                    [1., 2.1459/1.177], ['m', 'm'], ['-', '--']
+            ):
+                spline_contour = self.circular_contour(
+                    extra_ra_rad,
+                    extra_dec_rad,
+                    extra_radius_rad*cont_scale,
+                    healpy.get_nside(equatorial_map)
+                )
                 spline_lon = spline_contour[1]
                 spline_lat = np.pi/2. - spline_contour[0]
-                healpy.projplot(np.degrees(spline_lon), np.degrees(spline_lat), 
-                    lonlat=True, linewidth=2., color=cont_col, 
-                    linestyle=cont_sty)
+                healpy.projplot(
+                    np.degrees(spline_lon),
+                    np.degrees(spline_lat),
+                    lonlat=True,
+                    linewidth=2.,
+                    color=cont_col,
+                    linestyle=cont_sty
+                )
 
         plt.legend(fontsize=6, loc="lower left")
 
-
-        print("Contour Area (50%):", contour_areas[0], "degrees (scaled)")
-        print("Contour Area (90%):", contour_areas[1], "degrees (scaled)")
-
-
         # Dump the whole contour
-        pickle_path = self.output_dir / (unique_id + ".contour.pkl")
-        LOGGER.info(f"Saving contour to {pickle_path}")
-        with open(pickle_path, "wb") as f:
+        path = unique_id + ".contour.pkl"
+        print("Saving contour to", path)
+        with open(path, "wb") as f:
             pickle.dump(saving_contours, f)
 
-        healpy.write_map(self.output_dir / f"{unique_id}.skymap_nside_{mmap_nside}.fits.gz",
-            equatorial_map, coord = 'C', column_names = ['2LLH'],
-            extra_header = fits_header, overwrite=True)
+        # logic for multiordermaps -> for future developements
+
+        # save multiorder version of the map
+        # multiorder_map = mhealpy.HealpixMap(
+        #     grid_value, uniq_array
+        # )
+        # multiorder_map.write_map(
+        #     f"{unique_id}.skymap_nside_{mmap_nside}.multiorder.fits.gz",
+        #     column_names=["PROBABILITY"],
+        #     extra_header=fits_header,
+        #     overwrite=True,
+        # )
+
+        if llh_map:
+            column_names = ['2LLH']
+        else:
+            # avoid excessively heavy data format for the flattened map
+            equatorial_map = equatorial_map.clip(
```

Comment:
I tried using `.zst` but doesn't seem to solve the issue. Just tried with an event and after convolution, without the cut, the zst file was 100 MB. With the cut and `.gz` it is 1.5 MB, so an order of 100 smaller. I am not sure I am applying the zst compression as I should as even with the cut the zst file is still 100 MB.

Anyway, the cut is at `1e-16`, which even if all pixels where with a probability less than 1e-16 (surreal case), it would correspond to an addition of an order of `1e-9` to the whole probability. The 5 sigma level is at `1e-7` so even there the impact of this cut would be negligible, even more for the 3 sigma, and even more for the 50% and 90% contours. The renormalization is tu ensure the sum of all pixels equals 1, but it could also me removed, it is there just to make the result a bit nicer.